src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -24,6 +24,7 @@ const FORMNAME_CLASSNAME_MAPPINGS = {
     [AUTHENTICATOR_KEY.OV]: 'mfa-verify',
     [AUTHENTICATOR_KEY.GOOGLE_OTP]: 'mfa-verify',
     [AUTHENTICATOR_KEY.DUO]: 'mfa-verify-duo',
+    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: 'mfa-verify-symantec',
```

Comment:
Make sure we have that in v1

Reply:
I think this is called just `mfa-symantec` in V1. In that case we should add that instead ?

Reply:
Good catch! Re-mapped to `mfa-symantec`